lti-tool-provider/src/main/java/sikt/lti/tp/LtiLaunchResult.java

```diff
@@ -1,6 +1,9 @@
 package sikt.lti.tp;
 
-public class LtiLaunchResult {
+import no.unit.nva.commons.json.JsonSerializable;
+
+public class LtiLaunchResult implements JsonSerializable {
```

Comment:
Not needed to implement JsonSerializable?

Reply:
no need yes, my fault